src/expression/node/AccessorNode.js

```diff
@@ -47,9 +47,10 @@ export const createAccessorNode = /* #__PURE__ */ factory(name, dependencies, ({
      * @param {Node} object                 The object from which to retrieve
      *                                      a property or subset.
      * @param {IndexNode} index             IndexNode containing ranges
+     * @param {MetaOptions} object with additional options for building this node
```

Comment:
Just for clarity: to make this valid JSDoc, the lines like:
```
@param {MetaOptions} object with additional options for building this node
```
Should be changed to:
```
@param {MetaOptions} meta                     object with additional options for building this node
```
And if meta is optional:
```
@param {MetaOptions} [meta]                   object with additional options for building this node
```